frontend/src/components/projectDetailsHeader/ProjectDetailsHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import './ProjectDetailsHeader.css';
+import { Button, Menu, MenuItem, IconButton } from '@mui/material/';
+import { MoreHoriz, CloseRounded } from '@mui/icons-material/';
+import { useTaskStore, initialValue } from '../../store/task.store';
+
+const ProjectDetailsHeader: any = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const { setTask } = useTaskStore();
```

Comment:
Looks like you forgot to implement this, I cant see you using the project to setup the header

Reply:
I'm using setTask in order to close the Project details section.